healthtools/search/wit_ai.py

```diff
@@ -0,0 +1,60 @@
+"""
+Wit.ai will help predict the doc name the user is trying to search for. 
+This will be based on how the wit will have been trained. 
+wit.ai will be used when the query is made using /search?=<query>
+"""
+from wit import Wit 
+from nested_lookup import nested_lookup
+from healthtools.search import elastic, nurses
+from settings import WIT_ACCESS_TOKEN
+
+def determine_doc_type(query, doc_type=None):
+    """
+    This returns the doc name and the query.
+    The response will return 2 keys one being the doc name and the othere query
+    wit.ai returns all hyphens as underscores. 
+    """
+    client = Wit(access_token=WIT_ACCESS_TOKEN)
+    message_text = query
+    resp = client.message(message_text)
+    query = ''.join(nested_lookup('value', resp['entities']['query']))
+    doc_type = ''.join([var for var in (resp['entities'].keys()) if var != 'query'])
+    #wit returns the entity with an underscore,this changes it to hyphen
+    doc_type = doc_type.replace("_", "-")
+    return doc_type, query
+
+def find_search_type(doc_type):
+    """
+    This checks the doc type returned against the doc list an determines whether 
+    it should be elastic search or nurses search. If the doc type is empty,search type is None 
+    """
+    doc = ['nhif-outpatient', 'nhif-inpatient', 'nhif-outpatient-cs', 'doctors', 
+           'health-facilities', 'clinical-officers']
+    if doc_type not in  doc:
+        if doc_type is not 'nurses':
+            search_type = None
+        else:
+            search_type = 'nurses'
+    else:
+        search_type = 'elastic'
+    return search_type
```

Comment:
This is already defined in the `DOCUMENTS`. Why an entire function for it? https://github.com/CodeForAfricaLabs/HealthTools.API/blob/develop/healthtools/documents.py